lib/Sema/TypeCheckAttr.cpp

```diff
@@ -381,6 +383,13 @@ void AttributeChecker::visitNoImplicitCopyAttr(NoImplicitCopyAttr *attr) {
   }
 }
 
+void AttributeChecker::visitAlwaysEmitConformanceMetadataAttr(AlwaysEmitConformanceMetadataAttr *attr) {
```

Comment:
I think this kind of diagnostic usually comes for free from specifying the declaration types in the `SIMPLE_DECL_ATTR` declaration.

Reply:
Yup yup, thanks!